bool active
string target
int32 xline
int32 yline
float64 xgrid
float64 ygrid
float64 exposure_hot
float64 exposure_off
float64 exposure_on
string scan_direction

int32 current_line
string current_position

string from_node
float64 timestamp